### ISSUE TITLE
Add composer.json

### ISSUE THREAD

**Issue by nicodmf** [budgetinsight/budgea-clients#4]
To have a standard use of this package, can you add a composer.json (and certainly, a gem file to)

**Comment by maciejzgadzaj** [comment]
Good idea. Having the library listed on https://packagist.org/ would make it much easier to install and deploy it in projects.
